packages/patternfly-4/react-styled-system/scripts/copyTS.js

```diff
@@ -1,6 +1,8 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
 const path = require('path');
 const glob = require('glob');
 const fse = require('fs-extra');
+/* eslint-enable @typescript-eslint/no-var-requires */
```

Comment:
This is a build script. Not sure we need to apply linter here? Perhaps a different config for js?